packages/svelte/src/internal/client/dom/elements/attributes.js

```diff
@@ -355,22 +366,27 @@ var setters_cache = new Map();
 
 /** @param {Element} element */
 function get_setters(element) {
-	/** @type {string[]} */
-	var setters = [];
-	var descriptors;
-	var proto = get_prototype_of(element);
-
-	// Stop at Element, from there on there's only unnecessary setters we're not interested in
-	while (proto.constructor.name !== 'Element') {
-		descriptors = get_descriptors(proto);
-
-		for (var key in descriptors) {
-			if (descriptors[key].set && !always_set_through_set_attribute.includes(key)) {
-				setters.push(key);
+	var setters = setters_cache.get(element.nodeName);
+
+	if (!setters) {
+		setters = [];
+		setters_cache.set(element.nodeName, setters);
+
+		var descriptors;
+		var proto = get_prototype_of(element);
+
+		// Stop at Element, from there on there's only unnecessary setters we're not interested in
+		while (proto.constructor.name !== 'Element') {
```

Comment:
maybe this should reference the class directly? seems you could have `class Element extends HTMLElement {...}`

```suggestion
		while (proto.constructor !== Element) {
```